### PR TITLE
Everything can now be translated with locale file + French translation

### PR DESCRIPTION
**Pull request by Elinare**

### CODE REVIEW

File: pre-index.php [hunk 1]
```diff
@@ -595,15 +595,15 @@ class="onoffswitch-checkbox"/>
...
                         onclick="confirm('Are you sure you want to reset settings to default values?') ? (localStorage.clear(), window.location.reload()) : false">
-                    <i class="fa fa-refresh" aria-hidden="true"></i> Reset Settings
+                    <i class="fa fa-refresh" aria-hidden="true"></i> <?php echo i8ln('Reset Settings') ?>
                 </button>
             </center>
         </div>
         <div>
             <center>
                 <button class="settings"
-                        onclick="download('<?= addslashes($title) ?>', JSON.stringify(JSON.stringify(localStorage)))">
```
Comment (Glennmen) [inline pre-index.php]: why did you remove addslashes() function?

File: static/js/map.js [hunk 2]
```diff
@@ -2046,8 +2049,8 @@ function showGymDetails(id) { // eslint-disable-line no-unused-vars
 
             var raidStartStr = getTimeStr(result['raid_start'])
             var raidEndStr = getTimeStr(result['raid_end'])
-            raidStr += '<div>Start: <b>' + raidStartStr + '</b> <span class="label-countdown" disappears-at="' + result['raid_start'] + '" start>(00m00s)</span></div>'
-            raidStr += '<div>End: <b>' + raidEndStr + '</b> <span class="label-countdown" disappears-at="' + result['raid_end'] + '" end>(00m00s)</span></div>'
+            raidStr += '<div>' + i8ln('Start') + ': <b>' + raidStartStr + '</b> <span class="label-countdown" disappears-at="' + result['raid_start'] + '" start>(00m00s)</span></div>'
+            raidStr += '<div>' + i8ln('Fin') + ': <b>' + raidEndStr + '</b> <span class="label-countdown" disappears-at="' + result['raid_end'] + '" end>(00m00s)</span></div>'
```
Comment (Glennmen) [inline static/js/map.js]: This should be End

**Review by Glennmen — CHANGES_REQUESTED** [review]

File: pre-index.php (hunk 1 above)
Comment (Elinare) [reply pre-index.php]: Corrected

File: static/js/map.js (hunk 2 above)
Comment (Elinare) [reply static/js/map.js]: Corrected

**Review by Glennmen — APPROVED** [review]

**Review by Glennmen — CHANGES_REQUESTED** [review]
There are still text that has not been translated.
For example:
- Options in navbar
- In stats bar several things not translated
- Pokestop label

**Comment by Falaris01** [comment]
Thanks for your work, this is usually the first thing I merge when I start a new branch as I need to translate for my users aswell :)

Like someone else said, 10/10 would merge again  :D

**Review by Glennmen — APPROVED** [review]

File: static/js/stats.js [hunk 3]
```diff
@@ -133,19 +137,44 @@ function countMarkers(map) { // eslint-disable-line no-unused-vars
...
         pokestopListString += '</table>'
         document.getElementById('pokestopList').innerHTML = pokestopListString
     } else {
-        document.getElementById('pokestopList').innerHTML = 'PokéStops markers are disabled'
+        document.getElementById('pokestopList').innerHTML = i8ln('PokéStops markers are disabled')
     }
 }
+
+
+function i8ln(word) {
```
Comment (Glennmen) [inline static/js/stats.js]: What is the reason that you added this to stats.js?
The same function already exists in map.js
